hieradata_aws/class/jenkins.yaml

```diff
@@ -54,10 +54,18 @@ govuk_jenkins::config::banner_string: 'AWS Deploy'
 
 govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::job::deploy_app
+  - govuk_jenkins::job::deploy_emergency_banner
   - govuk_jenkins::job::deploy_puppet
+  - govuk_jenkins::job::deploy_router_data
   - govuk_jenkins::job::passive_checks
+  - govuk_jenkins::job::remove_emergency_banner
+  - govuk_jenkins::job::run_rake_task
+  - govuk_jenkins::job::search_benchmark
```

Comment:
In the configuration it uses: www-origin.integration.publishing.service.gov.uk, do we want to change this for AWS?

Reply:
Probably :) I just blindly added some jobs, good stuff in actually doing in the hard work and checking them!